src/pages/iou/steps/MoneyRequestAmountPage.js

```diff
@@ -394,9 +397,13 @@ class MoneyRequestAmountPage extends React.Component {
         // If a request is initiated on a report, skip the participants selection step and navigate to the confirmation page.
         if (this.props.report.reportID) {
             if (_.isEmpty(this.props.iou.participants)) {
+                const currentUserLogin = this.props.currentUserPersonalDetails.login;
                 const participants = ReportUtils.isPolicyExpenseChat(this.props.report)
                     ? [{reportID: this.props.report.reportID, isPolicyExpenseChat: true, selected: true}]
-                    : _.map(this.props.report.participants, (participant) => ({login: participant, selected: true}));
+                    : _.chain(this.props.report.participants)
+                        .filter((participant) => currentUserLogin !== participant.login)
+                        .map((participant) => ({login: participant, selected: true}))
+                        .value();
```

Comment:
Can you please clarify which bug is this fixing? And did we used to exclude self from participants?

Reply:
The bug:
1. Start a money request in an announce chat (make sure you have some members) and proceed to the confirmation page
<img width="378" alt="image" src="https://github.com/Expensify/App/assets/50919443/1cf589c0-5cf2-4079-83ce-2d69469342ce">

Notice my own user is duplicated.

We did filter it before:
https://github.com/Expensify/App/blob/9f1c0e08eb40f5703d12c9a62770509c70f78dff/src/pages/iou/MoneyRequestModal.js#L375

Reply:
Clear! mark as resolved